.github/workflows/playwright.yml

```diff
@@ -10,29 +10,28 @@ on:
         branches: [main, master]
     workflow_dispatch:
 
-env:
-    DATABASE_URL: ${{ vars.DATABASE_URL }}
-    E2E_USER_USERNAME: ${{ secrets.E2E_USER_USERNAME }}
-    E2E_USER_PASSWORD: ${{ secrets.E2E_USER_PASSWORD }}
-    KEYCLOAK_URL: ${{ vars.KEYCLOAK_URL }}
-    KEYCLOAK_REALM: ${{ secrets.KEYCLOAK_REALM }}
-    KEYCLOAK_CLIENT_ID_UI: ${{ secrets.KEYCLOAK_CLIENT_ID_UI }}
-    KEYCLOAK_CLIENT_SECRET_UI: ${{ secrets.KEYCLOAK_CLIENT_SECRET_UI }}
-    KEYCLOAK_CLIENT_ID_API: ${{ secrets.KEYCLOAK_CLIENT_ID_API }}
-    KEYCLOAK_CLIENT_SECRET_API: ${{ secrets.KEYCLOAK_CLIENT_SECRET_API }}
-    KEYCLOAK_ADMIN_USERNAME: ${{ secrets.KEYCLOAK_ADMIN_USERNAME }}
-    KEYCLOAK_ADMIN_PASSWORD: ${{ secrets.KEYCLOAK_ADMIN_PASSWORD }}
-    KEYCLOAK_ADMIN_USER_ID: ${{ secrets.KEYCLOAK_ADMIN_USER_ID }}
-    KEYCLOAK_ADMIN_CLIENT_SECRET: ${{ secrets.KEYCLOAK_ADMIN_CLIENT_SECRET }}
-    NEXTAUTH_URL: http://localhost:3000
-    NEXTAUTH_SECRET: ${{ secrets.NEXTAUTH_SECRET }}
-    NEXT_PUBLIC_API_URL: http://localhost:3001/api/
-
 jobs:
     test:
         timeout-minutes: 60
         runs-on: ubuntu-latest
         environment: test-environment
+        env:
```

Comment:
Commit message nit (optional):
- "VSCode provided" could be "VSCode-provided" or "autocomplete for secrets
doesn't work in VSCode"